ingestion/src/metadata/ingestion/source/database/clickhouse/queries.py

```diff
@@ -38,3 +38,25 @@
         LIMIT {result_limit}
 """
 )
+
+
+CLICKHOUSE_TABLE_COMMENTS = textwrap.dedent(
+    """
+SELECT database AS schema_name
+     , name AS table_name
```

Comment:
oh please move the commas to the line before `SELECT database AS schema_name,...`